src/main/java/ch/ivyteam/ivy/maven/IarPackagingMojo.java

```diff
@@ -47,7 +47,7 @@
 {
   public static final String GOAL = "pack-iar";
   private static final String[] DEFAULT_INCLUDES = new String[] {"**/*"};
-  private static final String[] DEFAULT_EXCLUDES = new String[] {"target", "target/**/*"};
+  private static final String[] DEFAULT_EXCLUDES = new String[] {"target", "%regex[target\\/(?!classes/).*]"};

```

Comment:
wow didn't know that regex statements are possible here. 
Thanks for this cool idea :)